src/platform/Linux/bluez/Helper.cpp

```diff
@@ -1547,6 +1547,11 @@ CHIP_ERROR InitBluezBleLayer(bool aIsCentral, char * apBleAddr, BLEAdvConfig & a
         err = ConfigureBluezAdv(aBleAdvConfig, endpoint);
         SuccessOrExit(err);
     }
+    else
+    {
+        // TODO: The "node id" means the bluetooth adapter on linux, this is confusing and should have a better name.
```

Comment:
this is very confusiong. Could we start with a separate rename that changes mNodeId to mAdapterId ?

Reply:
https://github.com/project-chip/connectedhomeip/pull/5173